src/systems/filecoin_blockchain/struct/block/election.go

```diff
@@ -12,6 +12,16 @@ func (tix *Ticket_I) ValidateSyntax() bool {
 	return tix.VRFResult_.ValidateSyntax()
 }
 
-func (tix *Ticket_I) Verify(proof util.Bytes, digest util.Bytes, pk filcrypto.VRFPublicKey) bool {
-	return tix.VRFResult_.Verify(proof, pk) && digest == blake2b.Sum256(proof)
+//func (tix *Ticket_I) Verify(proof util.Bytes, digest util.Bytes, pk filcrypto.VRFPublicKey) bool {
+//p := blake2b.Sum256(proof)
+//return tix.VRFResult_.Verify(proof, pk) && bytes.Compare(digest, p[:]) == 0
+//}
+
+func (tix *Ticket_I) Verify(randomness util.Bytes, pk filcrypto.VRFPublicKey, minerActorAddr addr.Address) bool {
```

Comment:
`Verify` func signature was being changed to the one commented out above, but there are a lot of places that actually refer to the previous signature, so I reverted it back (and left a commented out version of the new `Verify` at the top, in case we need to change this).